packages/webpack/package.json

```diff
@@ -28,7 +28,7 @@
     "access": "public"
   },
   "dependencies": {
-    "axios": "^0.21.1",
+    "axios": "^0.21.2",
```

Comment:
I'm going to send in a PR without this change, because we don't want to force anyone to upgrade.